.github/workflows/prepare_release.yml

```diff
@@ -0,0 +1,64 @@
+---
+
+name: Prepare Release
+
+on:
+  workflow_call:
+    inputs:
+      version:
+        description: 'Module version to be released.'
+        required: true
+        type: string
+      allowed_owner:
+        description: 'Only allow this owner'
+        required: true
+        type: string
+      working-directory:
+        description: The working directory where all jobs should be executed. Used for modules in subdirectories like a monorepo or a control repository.
+        default: '.'
+        required: false
+        type: string
+    secrets:
+      username:
+        description: 'The forge username to use'
+        required: true
```

Comment:
I don't see this secret used in the flow itself.

Reply:
dropped it